### ISSUE TITLE
Nullpointer on test run (loading of truststore fails)

### ISSUE THREAD

**Issue by MoritzKeppler** [FraunhoferISST/DataspaceConnector#19]
Test run fails, same applies for building docker container.
Build and run with -DskipTests works.

Details:
pathes to keystore and truststore are provided as URIs, e.g. `"@id" : "file:///conf/keystore-localhost.p12"`
KeyStoreManager extracts path but does not remove leading `/` as expected by `classLoader.getResourceAsStream(...)`

Stacktrace:
```
Caused by: java.lang.NullPointerException
	at de.fraunhofer.isst.ids.framework.configuration.KeyStoreManager.loadKeyStore(KeyStoreManager.java:105)
	at de.fraunhofer.isst.ids.framework.configuration.KeyStoreManager.<init>(KeyStoreManager.java:65)
	at de.fraunhofer.isst.ids.framework.spring.starter.ConfigProducer.<init>(ConfigProducer.java:52)
```

<sub> Moritz Keppler <moritz.keppler@daimler.com>, Daimler TSS GmbH, [legal info/Impressum](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)</sub>

**Comment by juliapampus** [comment]
The error can be reproduced on other Windows systems, but not on Linux. An issue to correct the file path handling was passed on to the framework developers. It will be fixed with an update of the framework version.

**Comment by hylkevds** [comment]
It happens on my Linux machine as well, on a fresh checkout followed by `mvn install`

**Comment by juliapampus** [comment]
@MoritzKeppler @hylkevds The error should now be fixed in the dev branch with Pull Request #31
